custom_components/bambu_lab/number.py

```diff
@@ -43,8 +43,8 @@ class BambuLabNumberEntityDescription(NumberEntityDescription, BambuLabTemperatu
         set_value_fn=lambda device, value: device.temperature.set_target_temp(TempEnum.NOZZLE, value)
     ),
     BambuLabNumberEntityDescription(
-        key="target_bed_temperature",
-        translation_key="target_bed_temperature",
+        key="bed_target_temperature",
+        translation_key="bed_target_temperature",

```

Comment:
Looks like this will break translations

Reply:
The problem is that these changes (much as I want to make them) break people's existing dashboards.